common/thorhelper/thorsoapcall.cpp

```diff
@@ -852,8 +861,13 @@ class CWSCHelper : implements IWSCHelper, public CInterface
             if ((flags & SOAPFliteral) && (flags & SOAPFencoding))
                 throw MakeStringException(0, "SOAPCALL 'LITERAL' and 'ENCODING' options are mutually exclusive");
 
-            header.set(s.setown(helper->getHeader()));
-            footer.set(s.setown(helper->getFooter()));
+            rowHeader.set(s.setown(helper->getHeader()));
```

Comment:
unrelated: Any idea why this is done this way, rather than header/footer etc. being OwnedRoxieStrings?